### ISSUE TITLE
subscription v2ray-json does not support quic

### ISSUE THREAD

**Issue by m0x61h0x64i** [Gozargah/Marzban#1086]
Hi, this quic inbound works for v2ray format, but does not work for v2ray-json format, i mean subscription cannot update

inbound:
```
    {
      "listen": "0.0.0.0",
      "port": 3636,
      "protocol": "vless",
      "settings": {
        "clients": [],
        "decryption": "none",
        "fallbacks": []
      },
      "sniffing": {
        "destOverride": [
          "tls",
          "quic",
          "fakedns"
        ],
        "enabled": true,
        "metadataOnly": false,
        "routeOnly": false
      },
      "streamSettings": {
        "network": "quic",
        "quicSettings": {
          "header": {
            "type": "none"
          },
          "key": "key",
          "security": "none"
        },
        "security": "none"
      },
      "tag": "inbound-3636"
    },
```

marzban logs:
![image](https://github.com/Gozargah/Marzban/assets/109550653/1451d8f7-4c9b-4e0c-8e21-00bd8b4889a1)
![image](https://github.com/Gozargah/Marzban/assets/109550653/43893bd4-a675-4c2b-87d9-0da92d9724c8)

**Comment by M03ED** [comment]
hey , what version you are using ?
i think we fix this few days ago #961

**Comment by m0x61h0x64i** [comment]
im using this commit a2ad3171eeebc8308027578eaedd956deff4caf8 which is for 2 days ago
